README.md

```diff
@@ -91,12 +108,10 @@ Quicklinks:
  - [🔨 Rejected rust we want to support](https://github.com/hacspec/hax/issues?q=is%3Aissue+is%3Aopen+label%3Aunsupported-rust+-label%3Awontfix%2Cwontfix-v1);
  - [💭 Rejected rust we don't plan to support in v1](https://github.com/hacspec/hax/issues?q=is%3Aissue+is%3Aopen+label%3Aunsupported-rust+label%3Awontfix%2Cwontfix-v1).
 
-## Examples
-
-There's a set of examples that show what hax can do for you.
-Please check out the [examples directory](examples/).
-
 ## Hacking on Hax
+The documentation of the internal crate of hax and its engine can be
```

Comment:
We should publish the hax-lib docs there as well and link them in the readme. Do we have an issue for that?

Reply:
Yes, I agree, that'd be good. No, let me add one!

Reply:
https://github.com/hacspec/hax/issues/658